lib/ansible_tower_client/base_models/project_update.rb

```diff
@@ -1,4 +1,8 @@
 module AnsibleTowerClient
   class ProjectUpdate < BaseModel
+    def stdout(format = 'txt')
+      api.get("#{related['stdout']}?format=#{format}").body
+    end
+    alias result_stdout stdout
```

Comment:
Does this need to be a version dependent patch?  Will this break older versions?

Reply:
no. I checked and this is available since 3.1

Reply:
Do we still support versions older than 3.1?

Reply:
Not sure.
At least 3.0 reached End of Maintenance Support 2 (EOL) on 2/28/18 according to 
https://access.redhat.com/support/policy/updates/ansible-tower

We started supporting Embedded Tower since 3.1. I am not sure about the external integration. Do you remember what version of Tower is the first integrated by MIQ?

Reply:
Since this change is intended to fix master and hammer branches I think we should be ok here based on the Tower EOL for 3.0.